ui/raidboss/data/triggers/o7s.js

```diff
@@ -48,107 +102,199 @@
       condition: function(data, matches) { return matches[1] == data.me; },
-      alertText: 'Orb Marker',
-      tts: 'orb',
+      alertText: function(data) {
+		  return {
+			  en: 'Orb Marker',
+			  de: 'Orb Marker'
+		  };
+	  },
+      tts: function(data) {
+		  return {
+			  en: 'orb',
+			  de: 'orb marker',
+		  };
+	  },
     },
     {
       id: 'O7S Blue Marker',
       regex: /1B:........:(\y{Name}):....:....:000E:0000:0000:0000:/,
       alarmText: function(data, matches) {
         if (data.me == matches[1])
-          return 'Blue Marker on YOU';
+          return {
+			  en: 'Blue Marker on YOU',
+			  de: 'Aura-Kanone auf DIR',
+		  };
       },
       infoText: function(data, matches) {
         if (data.me != matches[1])
-          return 'Blue Marker on ' + data.ShortName(matches[1]);
+          return {
+			  en: 'Blue Marker on ' + data.ShortName(matches[1]),
+			  de: 'Aura-Kanone auf ' + data.ShortName(matches[1]),
+		  };
       },
-      tts: function (data, matches) {
+      tts: function(data, matches) {
         if (data.me == matches[1])
-          return 'blue marker';
+          return {
+			  en: 'blue marker',
+			  de: 'aura-kanone',
+		  };
       },
     },
     {
       id: 'O7S Prey',
       regex: /1B:........:(\y{Name}):....:....:001E:0000:0000:0000:/,
       infoText: function(data, matches) {
         if (data.me == matches[1])
-          return 'Prey on YOU';
-        return 'Prey on ' + data.ShortName(matches[1]);
+          return {
+			  en: 'Prey on YOU',
+			  de: 'Rakete auf dir'
+		  };
+        return {
+			en: 'Prey on ' + data.ShortName(matches[1]),
+			de: 'Rakete auf ' + data.ShortName(matches[1]),
+		};
       },
-      tts: function (data, matches) {
+      tts: function(data, matches) {
         if (data.me == matches[1])
-          return 'prey';
+          return {
+			  en: 'prey',
+			  de: 'rakete',
+		  };
       },
     },
     {
       id: 'O7S Searing Wind',
-      regex: / 1A:(\y{Name}) gains the effect of Searing Wind/,
+      regex: / 1A:(\y{Name}) gains the effect of (?:灼熱|Searing Wind|Gluthitze|Fournaise)/,
       condition: function(data, matches) { return data.me == matches[1]; },
-      alarmText: 'Searing Wind: go outside',
-      tts: 'searing wind',
+      alarmText: function(data) {
+		  return {
+			  en: 'Searing Wind: go outside',
+			  de: 'Gluthitze: Geh weg',
+		  };
+	  },
+      tts: function(data) {
+		  return {
+			  en: 'searing wind',
+			  de: 'gluthitze',
+		  };
+	  },
     },
     {
       id: 'O7S Abandonment',
-      regex: / 1A:(\y{Name}) gains the effect of Abandonment/,
+      regex: / 1A:(\y{Name}) gains the effect of (?:孤独感|Abandonment|Verlassen|Isolement)/,
       condition: function(data, matches) { return data.me == matches[1]; },
-      alertText: 'Abandonment: stay middle',
-      tts: 'abandonment',
+      alarmText: function(data) {
+		  return {
+			  en: 'Abandonment: go mid',
```

Comment:
Please use spaces instead of tabs, here and elsewhere.

Reply:
seems like even if you put notepad++ to space instead of tabs, the spaces are still tabs in disguise..